src/docfx/build/DocsetBuilder.cs

```diff
@@ -180,7 +180,7 @@ public void Build(string[]? files)
                     () => _repositoryProvider.Save(),
                     () => _errors.AddRange(_githubAccessor.Save()),
                     () => _errors.AddRange(_microsoftGraphAccessor.Save()),
-                    () => _jsonSchemaTransformer.PostValidate(),
+                    () => _jsonSchemaTransformer.PostValidate(files != null),
```

Comment:
how about sharing the logic with line 165

Reply:
I'd love to, but the nullability flow analysis isn't smart enough to know that when `isPartialBuild` is `true`, `files` is non-nullable

![image](https://user-images.githubusercontent.com/511355/103610694-b3acda00-4f5b-11eb-9abe-d85c8a9ef7fa.png)

Reply:
https://github.com/dotnet/roslyn/issues/50250